src/test/java/car/RacingCarTest.java

```diff
@@ -21,16 +27,41 @@ public void testIsMoving(int input, boolean output) {
     }
 
     @Test
-    @DisplayName("게임 로직 정상실행 통합 테스트")
-    public void testGameLogic() {
-        RacingCarGameLogic.getInstance().gameLogic(3,5);
+    @DisplayName("자동차 이름 초기화 확인 테스트")
+    public void testSaveCarName() {
+        String[] names = TEST_CAR_NAMES.split(",");
+        List<Car> cars = RacingCarGameLogic.getInstance().initCars(TEST_CAR_NAMES);
+        IntStream.range(0, names.length).forEach(i -> assertThat(cars.get(i).getName()).isEqualTo(names[i]));
+    }
+
+    @Test
+    @DisplayName("3명인 경우 우승자 거리가 0인 경우 3명 공동 우승 확인 테스트")
+    public void testGetZeroDistanceWinner() {
+        List<Car> cars = Arrays.asList(new Car("test0", 0), new Car("test1", 0), new Car("test2", 0));
+        List<Car> winners = RacingCarGameRule.getInstance().getWinnerCars(cars);
+        assertThat(winners.size()).isEqualTo(3);
+    }
+
+    @Test
+    @DisplayName("3명인 경우 1명의 우승자 구하는 확인 테스트")
+    public void testGetWinner() {
+        List<Car> cars = IntStream.range(0, 3).mapToObj(index -> new Car("test" + index, index)).collect(Collectors.toList());
+        List<Car> winners = RacingCarGameRule.getInstance().getWinnerCars(cars);
+        assertThat(winners.get(0).getName()).isEqualTo("test2");
```

Comment:
```suggestion
        assertThat(winners.get(0)).isEqualTo(new Car("test2", 2));
```
이렇게도 테스트하면 좋을 것 같은데 ...
고민해 보면 좋겠습니다! 😃